saphanabootstrap-formula.spec

```diff
@@ -32,21 +35,20 @@ Requires:       habootstrap-formula
 Requires:       salt-shaptools
 Suggests:       hanadb_exporter >= 0.3.1
 
-# On SLE/Leap 15-SP1 and TW requires the new salt-formula configuration location.
-%if ! (0%{?sle_version:1} && 0%{?sle_version} < 150100)
```

Comment:
This cannot be dropped, as this package already exists on TW, so the automatic formula configuration is already possible there. Also, having this dependency helps to avoid file conflicts and dependency problems between the packages. It was only removed on the PackageHub submission as this package is not yet available there. As soon as it gets included there, the same TW package will be synced on PackageHub.

Reply:
So, in this case, we will manage the packageHub spec file manually, right?

Reply:
Yes, only for now.